payments/models.py

```diff
@@ -105,7 +124,12 @@ class Rate(models.Model):
     price = models.DecimalField(max_digits=6, decimal_places=2,
                                 verbose_name='price (USD)')
     capacity = models.PositiveIntegerField()
-
+    discount = models.BooleanField(default=False)
+    discount_code = models.SlugField(help_text='This will be the code given to'
+                                     'a customer receiving a discount in the '
+                                     'form of https://workshops.qiime.org/wor'
+                                     'kshop_slug/rate=discount_code',
+                                     default=uuid.uuid4)
```

Comment:
That seems to be a Django issue. Probably something weird thing to do with the inline admin. I'll do some digging to see if there is any way to fix it.